addons/main/XEH_preInit.sqf

```diff
@@ -211,6 +211,7 @@ _curCat = localize "STR_dui_cat_custom_color";
 if !(hasInterface) exitWith {};
 
 GVAR(radioModSpectator) = configFile call {
+    if (isClass (_this >> "CfgPatches" >> "task_force_radio")) exitWith { {_player getVariable ["tf_forceSpectator", false]} };
```

Comment:
This is wrong, this will return tf_forceSpectator even for TFAR beta which uses TFAR_forceSpectator

Reply:
so I should move it under TFAR beta check